index.js

```diff
@@ -32,6 +37,59 @@ module.exports = {
         ],
       },
     },
+    delete: {
+      validation: {
+        function: 'checkboxValidation',
+        on: 'change',
+      },
+      script: {
+        function: 'fileScript',
+        on: 'change',
+      },
+      type: 'checkbox',
+      label: 'Delete file',
+      options: [
+        { label: 'Delete file',
+          value: 'true',
+        },
+      ],
+      settings: {
+        classes: {
+          hide: 'file--uploaded',
+        },
+      },
+    },
   },
-  html: '<label for="{{file.id}}">{{file.label}}</label><input type="{{file.type}}" id="{{file.id}}" name="{{file.name}}" value="{{file.value}}" {% if settings.types %}{%set comma = joiner() %}accept="{% for type in settings.type %}{{comma()}}{{type}}{% endfor %}{% endif %}" />',
+  html: `<label for="{{file.id}}">{{file.label}}</label>
+    {% set classes = '' %}
+    {% set filepath = '/files' %}
```

Comment:
This doesn't belong here. The input plugin shouldn't be making assumptions as to where a file is stored, it should just get the file object, which includes the absolute path of the file and the and the file type.